Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -35,6 +35,52 @@ example, if the Rook operator is running in the namespace `my-namespace` the
 provisioner value should be `my-namespace.rbd.csi.ceph.com`. The same provisioner
 name must be set in both the storageclass and snapshotclass.
 
+## Configure custom Driver name prefix for CSI Drivers
+
+To use a custom prefix for the CSI drivers, you need to set the
```

Comment:
Let's mention that the namespace is the default prefix
```suggestion
To use a custom prefix for the CSI drivers instead of the namespace prefix, set the
```